README.md

```diff
@@ -4,6 +4,22 @@ Public scripts to launch near devnet, betanet and testnet node
 
 # Usage
 
+## Prerequisite
+
+Before you proceed, make sure you have the following software installed:
+
+* Python 3
+* git (used for updates)
+* cURL (installation only)
```

Comment:
git and curl seems builtin in most system, but this gives absolutely guarantee :+1:

Reply:
These are reported by a real user trying to run nearup on a fresh Google Cloud instance. Well, only `git` was missing, but I decided to be explicit.